### PR TITLE
fix a argument error

### PR DESCRIPTION
**Pull request by ranchlai**
# What does this PR do?

This PR fixes the error when pad_token_id is set ( in order to user batch size > 1). 
```bash 
>>>  File "optimum/optimum/gptq/data.py", line 57, in <listcomp>
collate_data(new_examples[start : start + batch_size], pad_token_id)
  File "optimum/optimum/gptq/data.py", line 88, in collate_data
    label_blocks = [block["labels"] for block in blocks]
  File "optimum/optimum/gptq/data.py", line 88, in <listcomp>
    label_blocks = [block["labels"] for block in blocks]
KeyError: 'labels'
```
@SunMarc  Please have a look

### CODE REVIEW

**Comment by ranchlai** [comment]
I just noticed that there is already an issue here #1447 .

**Review by SunMarc — APPROVED** [review]
LGTM ! Thanks for the fix !

**Review by fxmarty — APPROVED** [review]
LGTM

**Comment by fxmarty** [comment]
@ranchlai? Before merging, could you run `make style`? I don't have push permission to your branch unfortunately.

**Comment by ranchlai** [comment]
Sure. I have done that. This will reformat anther file though. @fxmarty

**Pull request merged**
